.gitignore

```diff
@@ -13,3 +13,4 @@ bash.exe.stackdump
 auth/keycloak/config/hsperfdata_jboss/
 auth/keycloak/config/*.log
 trufflehog_report.*
+datastore.xml
```

Comment:
this is a workaround.

geoserver supports env var substitution in its datastore files:
https://docs.geoserver.org/master/en/user/datadirectory/configtemplate.html

however, I could not get this to work in the datastore.xml for the mssql connection, possibly because this datastore is specific to the optional sql-server-plugin and not part of the core geoserver functionality

Reply:
so instead, ignore this file as each developer will have their own unique connection string. Instead, add this file to the files generated by gen-env-files.sh